cf-serverd/cf-serverd-functions.c

```diff
@@ -540,14 +555,71 @@ static void PrepareServer(int sd)
     WritePID("cf-serverd.pid"); /* Arranges for cleanup() to tidy it away */
 }
 
+/* Graceful Stop
+ * This runs a new main process that will die and that init can restart (as systemd can do).
+ * This this can prevent systemd from kill us it there is a problem.
+ * But this makes it possible to finish kandling connections while systemd tries to restart us.
+ * If there is no systemd make sure alternative init restarts us. */
+static void GracefulStop()
+{
+    Log(LOG_LEVEL_ERR, "Stopping gracefully");
```

Comment:
probably not LOG_LEVEL_ERR, we should have one info log and the rest in verbose or debug.